conn.go

```diff
@@ -40,22 +53,33 @@ func (c *Conn) WriteUnreliablePacket(buf []byte) error {
 }
```

Comment:
warning: exported method Conn.Close should have comment or be unexported (golint)